Dockerfile

```diff
@@ -11,13 +11,19 @@ COPY . .
 # Fetch dependencies
 RUN dep ensure --vendor-only
 
-# Build binary
-RUN ./configure && make build
+# Build binary and make sure there is at least an empty key file.
```

Comment:
We need a file to be there, otherwise the COPY of it later on will fail and bomb out the docker build.  If you have a better way to do this, I'm all ears!